Include <unistd.h> to fix:

RtAudio.cpp:1128:21: error: use of undeclared identifier 'getpid'
    if ( hog_pid != getpid() ) {
                    ^
RtAudio.cpp:1129:17: error: use of undeclared identifier 'getpid'
      hog_pid = getpid();
                ^
RtAudio.cpp:1177:7: error: use of undeclared identifier 'usleep'
      usleep( 5000 );
      ^
3 errors generated.
--- src/RtAudio.cpp.orig	2019-04-18 08:27:12.000000000 -0500
+++ src/RtAudio.cpp	2020-07-03 06:40:40.000000000 -0500
@@ -48,6 +48,7 @@
 #include <climits>
 #include <cmath>
 #include <algorithm>
+#include <unistd.h>
 
 // Static variable definitions.
 const unsigned int RtApi::MAX_SAMPLE_RATES = 14;
